cmd/podmon/Dockerfile

```diff
@@ -1,4 +1,6 @@
 FROM registry.access.redhat.com/ubi8:8.3
 RUN dnf -y install openssl
+RUN dnf -y install nettle
```

Comment:
The image layer count could be minimized here by collapsing these 3 RUN statements into 1.

Reply:
Sounds good. I'll update it.